RecoEgamma/EgammaIsolationAlgos/plugins/EgammaPhotonTkNumIsolationProducer.cc

```diff
@@ -6,18 +6,40 @@
 //=============================================================================
 //*****************************************************************************
 
-#include "RecoEgamma/EgammaIsolationAlgos/plugins/EgammaPhotonTkNumIsolationProducer.h"
-
-// Framework
-#include "DataFormats/Common/interface/Handle.h"
-
-#include "DataFormats/Candidate/interface/Candidate.h"
-#include "DataFormats/Candidate/interface/CandAssociation.h"
-#include "DataFormats/TrackReco/interface/TrackFwd.h"
```

Comment:
The `Track` class is still needed, probably `Track.h` was included indirectly before. This was again removed because of me disliking included of `*Fwd.h` in `.cc` files, but I'll put it back.